tasks/tasks.py

```diff
@@ -122,10 +122,10 @@ def upload_artifacts(self):
                             pr_author=self.pr_author,
                             task_name=self.task_name,
                             returncode=self.returncode,
-                            timeout=5*60))
+                            timeout=5 * 60))
         except Exception as exc:
             logging.debug(exc, exc_info=True)
-            hostname = socket.gethostname().split('.')[0] # don't leak fqdn
+            hostname = socket.gethostname().split('.')[0]  # don't leak fqdn
```

Comment:
I think your editor wants to lint this file. Could you please remove this (and others) from the PR.